sphinx_gallery/tests/tinybuild/examples/__init__.py

```diff
@@ -0,0 +1 @@
+from numpy import pad
```

Comment:
Maybe add a comment here rather than an unused import? Something like this for example?

```py
# This file is ignored through ignore_pattern so it does not need to have a
# docstring following the sphinx-gallery convention (i.e. title + description)
```